src/main/groovy/com/okta/test/mock/application/ApplicationTestRunner.groovy

```diff
@@ -62,6 +62,10 @@ abstract class ApplicationTestRunner extends HttpMock {
         return mockPort
     }
 
+    int doGetMockHttpsPort() {
```

Comment:
Rename to `getMockHttpsPort()`?

Reply:
`doGetMockHttpsPort()` is actually called by a method named `getMockHttpsPort()`, which caches the result allowing `getMock[Https]Port()` to be called multiple times, and the actual implementation `doGetMock[Https]Port()` doesn't need to worry about the state.  I'll add a couple comments around this to make it more clear!